calc.feature

```diff
@@ -1,21 +1,22 @@
 Feature: window Calculator
 
+Scenario: Open calculator
+    * robot { window: 'Calculator', fork: 'calc' }
+    * call read('calc.feature@tag1')
+    * click('Close Calculator')
+
+
+@tag1 @ignore
 Scenario Outline: automate the addition, subtraction, multiplication and division of any two given number.
     * def num1 = 'Six'
     * def num2 = 'Two'
```

Comment:
you can move this to above scenario. We don't need to define it each and every time

Reply:
okay. I will change it.